digital_land/cli.py

```diff
@@ -65,6 +69,15 @@ def issue_path(f):
     )(f)
 
 
+def source_endpoint_dir(f):
+    return click.option(
+        "--source-endpoint-dir",
```

Comment:
I _think_ this would be better specifying the two files separately as `endpoint-path` and `source-path`. We already use `endpoint-path` for the collect action so can share the option definition with that. Willing to have my mind changed though!

Reply:
I debated myself with the same question (I was wondering whether the two files would ever live in different directories). But two paths is more flexible anyway, I'll do that 👍